cssFiles/cardfiles/cardswithbadges.css

```diff
@@ -0,0 +1,111 @@
+.container1{
+    position: relative;
+    border: 1rem;
+    margin: 1rem;
+    width: 250px;
+    border-radius: 10px;
+    
+  }
+  
+  .ecom-img1{
+    position: relative;
+    height: 15rem;
+    width: 100%;
+    border-radius: 5px 5px 5px 5px;
+    
+  }
+  
+  .secondary-container1{
+    display: flex;
+    /* gap: 15px; */
+    
+  }
+  
+  .actual-price1 {
+      display: inline;
+      margin-right: 4px;
+      color: grey;
+      text-decoration: line-through;
+    }
+    .disc-price1 {
+      margin-right: 4px;
+      display: inline;
+    }
+    .offer-info1 {
+      margin-right: 4px;
+      display: inline;
+      color: darkgreen;
+    }
+  .card-footer1 {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      margin-top: auto;
+      justify-content: center;
+      gap: 1rem;
+    }
+    .btn-link1 {
+      background: transparent;
+      border: 0;
+      font-size: 1.2rem;
+      padding: 5px 10px;
+      border-radius: 5px;
+    }
+    .call-to-action1 {
+      margin-top: 1rem;
+      background-color: darkorange;
+      color: var(--white);
+    }
+    .call-to-action1:hover {
+      background-color: orange;
+    }
+  
+    .vertical-btn1{
+      width: 100%;
+    }
+  
+     .card-badge1 {
+      z-index: 2;
+      position: absolute;
+      left: .025rem;
+      background: orange;
+      color: var(--lynx-white);
+      padding: 5px;
+      font-size:smaller
+    }
+  
+     .heart-badge1 {
+      z-index: 2;
+      position: absolute;
+      right: .03rem;
+      color:red;
+      padding: 5px;
+      
+    }
+
+    .cardnew{
+        min-height: 100px;
```

Comment:
If you're using `rems` in your project, try to use them whenever necessary as they are also responsive. You can use `px to rem` converters online if you have a specific size in mind(in px) and want to convert it to rem